admin-dev/themes/default/template/controllers/translations/helpers/view/main.tpl

```diff
@@ -197,7 +197,7 @@
 			</div>
 		</div>
 	</form>
-	<form action="{$url_submit|escape:'html':'UTF-8'}" method="post" enctype="multipart/form-data" class="form-horizontal">
+	<form action="index.php/international/translations/extract" method="post" enctype="multipart/form-data" class="form-horizontal">
```

Comment:
hmm, i'm not sure if we should use hardcoded action? do we really cant use some route helper from Twig?